src/tracer.js

```diff
@@ -96,7 +100,18 @@ export default class Tracer {
     this.registerInjector(opentracing.FORMAT_BINARY, binaryCodec);
     this.registerExtractor(opentracing.FORMAT_BINARY, binaryCodec);
 
-    this._reporter.setProcess(this._serviceName, Utils.convertObjectToTags(this._tags));
+    const uuid = Utils.getRandom64().toString('hex');
+    this._tags[constants.TRACER_CLIENT_ID_TAG_KEY] = uuid;
+    this._process = {
+      serviceName: serviceName,
+      tags: Utils.convertObjectToTags(this._tags),
+      uuid: uuid,
+    };
+    if (typeof this._debugThrottler.setProcess === 'function') {
```

Comment:
The default throttler can have `.setProcess(...)` as a noop. This would keep the interface consistent. Same for `.close(...)`.